live/src/fix_bootconfig

```diff
@@ -13,12 +13,18 @@
 #    This script (a) adjusts the boot config, (b) runs xorriso, and (c) makes
 #    it zIPL bootable on s390x.
 #
+set -x
 
-# KIWI config
-test -f .profile && . .profile
 
 dst=${1#iso:}
 
+# KIWI config
```

Comment:
It's important to do this after setting $dst. I was not able to get acccess to $kiwi_* variables (set in /.profile) fix_bootconfig.$arch . So I simply copied the .profile into the $dst from where the xorriso -> fix_bootconfig$ arch was executed.

This was working (vars inherited, without copying the file) if the fix_bootconfig was called outside xorriso.